specs/capella/beacon-chain.md

```diff
@@ -226,6 +241,8 @@ class BeaconState(Container):
     # Withdrawals
     next_withdrawal_index: WithdrawalIndex  # [New in Capella]
     next_withdrawal_validator_index: ValidatorIndex  # [New in Capella]
+    # Deep history valid from Capella onwards
+    historical_batches: List[HistoricalBatchSummary, HISTORICAL_ROOTS_LIMIT]  # [New in Capella]
```

Comment:
I might just call this `historical_summaries` and `HistoricalSummary`. 
slight preference for the var name to match the class name

Reply:
No objections